.circleci/config.yml

```diff
@@ -1,91 +0,0 @@
-version: 2
-
-experimental:
-  notify:
```

Comment:
This was the only part I was unsure about, I am not sure what the notify is doing